### PR TITLE
initial impl

### PR DESCRIPTION
**Pull request by pavolloffay**
@yurishkuro @objectiser could you please review?

### CODE REVIEW

**Comment by yurishkuro** [comment]
cc @black-adder

Do we want to cross-test this as well? I.e. run the crossdock test here using latest test image against the current build of zipkin-brave test?

**Comment by pavolloffay** [comment]
Once this is approved/merged I will just move the repository to `jaegertracing`. I would like to also setup an automated build in jaegertracing docker hub.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+COPY .mvn $APP_HOME/.mvn
+COPY mvnw $APP_HOME
+
+WORKDIR $APP_HOME
+RUN ./mvnw package
+
+EXPOSE 8080
+EXPOSE 8081
+
+CMD ["java", "-jar", "target/jaegertracing-xdock-brave-0.0.1-SNAPSHOT.jar"]
```
Comment (objectiser) [inline Dockerfile]: Be better to use finalName in pom to remove the version, so don't need to keep updating this file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Crossdock Zipkin Brave
+
+This is a [crossdock](https://github.com/uber/jaeger/tree/master/crossdock) app to test Zipkin data format support in Jaeger server. 
+It uses Brave to create spans which are reported to the server.
+
+## Build Docker image
+Docker image is automatically build once commit is pushed to master.
```
Comment (objectiser) [inline README.md]: nit: built

File: src/main/java/io/jaegertracing/xdock/zipkin/EndToEndBehaviourController.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package io.jaegertracing.xdock.zipkin;
```
Comment (objectiser) [inline src/main/java/io/jaegertracing/xdock/zipkin/EndToEndBehaviourController.java]: JaegerAuthors copyright header - and on other files.

File: Dockerfile (hunk 1 above)
Comment (pavolloffay) [reply Dockerfile]: There is no plan to release it for now. I will keep this for simplicity.

**Comment by pavolloffay** [comment]
@objectiser here you can find some info https://github.com/crossdock/crossdock. This framework just executes rest endpoint on `test_driver` with parameters from matrix then `test_driver` calls other services and verifies tests.

**Comment by pavolloffay** [comment]
I have done some minor improvements (reduce image size, polish the code..). I will merge and move it to the `jaegertracing`

**Comment by pavolloffay** [comment]
@yurishkuro I think it's good to add xdock also here (to prevent failures in the main repo), however https://github.com/uber/jaeger/pull/378 has to go in before (enables zipkin collector in test_driver).

**Pull request merged**